src/gsy_e/gsy_e_core/export.py

```diff
@@ -167,7 +167,7 @@ def area_tree_summary_to_json(self, data: Dict) -> None:
         subdirectory = pathlib.Path(self.directory, "raw_data")
         if not subdirectory.exists():
             subdirectory.mkdir(exist_ok=True, parents=True)
-        json_file = os.path.join(self.directory, "area_tree_summary.json")
+        json_file = os.path.join(subdirectory, "area_tree_summary.json")
```

Comment:
@spyrostz this change will move the `area_tree_summary.json` from the root results folder into the `raw_data` folder. Before I adapt the integration tests. Is this where it supposed to be or was the creation of the `raw_data` path in this method misleading and it should indeed live in the root folder?

Reply:
Hmmm, in this case I think that it would be better to revert the change. This file is used as a reference for grid tree information, which IMO are not considered raw data, but rather configuration. Therefore I think it would be better to keep on living on the root directory.